tesseract/tesseract_planning/tesseract_process_managers/src/taskflow_generators/graph_taskflow.cpp

```diff
@@ -182,8 +182,8 @@ tf::Taskflow& GraphTaskflow::generateTaskflow(ProcessInput input,
       {
         throw std::runtime_error("Invalide Edges for process index: " + std::to_string(src_idx));
       }
-      ++src_idx;
     }
+    ++src_idx;
```

Comment:
Ah. I noticed that TASK nodes didn't seem to work before. I just made it a conditional with both conditions going to the same node as a workaround. Good find.

Reply:
Yea it took a while to figure out what was going on. I was getting weird graphs.